students/tlugosan/lesson07/html_render.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+
+
+class Element(object):
+    tag_name = "html"
+    indent = '    '
+
+    def __init__(self, content=None, **kwargs):
+        self.kwargs = ""
+        # style="text-align: center; font-style: oblique;"
+        if self.kwargs != {}:
+            for k in kwargs:
+                self.kwargs += ' ' + str(k) + '=' + '"' + str(kwargs[k]) + '"'
+        if content is None:
+            self.content = []
+        else:
+            self.content = [content]
+
+    def append(self, new_content):
+        self.content.append(new_content)
+
+    def render(self, file_out, cur_ind=''):
+        cur_ind += self.indent
+        open_tag = "<{}{}>\n".format(self.tag_name, self.kwargs)
+        end_tag = "</{}>".format(self.tag_name)
+        file_out.write(cur_ind + open_tag)
+        for each_content in self.content:
+            try:
+                each_content.render(file_out, cur_ind)
+            except AttributeError:
+                file_out.write(self.indent + cur_ind + str(each_content))
+            file_out.write('\n')
+
+        file_out.write(cur_ind + end_tag)
+
+
+class Html(Element):
+    tag_name = 'html'
+
+    def __init__(self, content=None, **kwargs):
+        super().__init__(content, **kwargs)
+
+    def render(self, file_out, cur_ind=''):
+        file_out.write('<!DOCTYPE html>\n')
+        open_tag = "<{}{}>\n".format(self.tag_name, self.kwargs)
+        end_tag = "</{}>".format(self.tag_name)
+        file_out.write(cur_ind + open_tag)
+        for each_content in self.content:
+            try:
+                each_content.render(file_out)
+            except AttributeError:
+                file_out.write(str(each_content))
+            file_out.write('\n')
+
+        file_out.write(cur_ind + end_tag)
+
+
+class Body(Element):
+    tag_name = 'body'
+
+
+class P(Element):
+    tag_name = 'p'
+
+
+class Head(Element):
```

Comment:
note that for python3 you don't need to extend from `object`; `class Head:` is sufficient